plf-packaging-resources/src/main/resources/README.txt

```diff
@@ -9,13 +9,13 @@ System requirements
     * Memory:  4GB of RAM (8GB recommended)
     * Disk:    2GB (depending of the amount of data)
     * OS:      Windows or Linux
-    * JDK:     Java 7 (Set the JAVA_HOME environment variable)
-    * Browser: Google Chrome 25+, Firefox 19+ or Internet Explorer 10+
+    * JDK:     Java 8 (Set the JAVA_HOME environment variable)
+    * Browser: Google Chrome 62+, Firefox 52.4+ or Internet Explorer 11+
```

Comment:
"Edge 15" (as recommended) and "Safari 10.1" as (supported)

Reply:
IE11 is still supported. And we do not support Safari (only Compatible versions).

Reply:
> IE11 is still supported

I know, but Edge is a different browser, thus it should be added in the list.

> we do not support Safari

it's supported for iOS

Reply:
> I know, but Edge is a different browser, thus it should be added in the list.

Ok I thought you wanted to replace IE11 by Edge. Done.

> it's supported for iOS

Not on desktop so. We should probably add a new item to distinguish desktop and mobile browsers. WDYT ?

Reply:
Good idea (y)
May be adding:
* Browser (desktop and tablet) : ...
* Browser (Mobile) : ...

Reply:
Done